selectors-4/Overview.bs

```diff
@@ -2256,20 +2256,9 @@ The Focus-Indicated Pseudo-class: '':focus-visible''</h3>
 		like clicking on the button with a mouse pointer.
 	</div>
 
-	<div class=advisement>
-		Page authors should follow these guidelines
```

Comment:
It seems a little disappointing to remove all of the advice for authors.  It's not clear to me that the old advice was the best that could be offered, but this does seem like an area where it's worth offering some expert advice on what the right thing really is.